src/index.js

```diff
@@ -56,6 +56,8 @@ app.ports.writeToLocalStorage.subscribe(([key, value]) => {
   localStorage.setItem(key, value)
 })
 
+app.ports.openInNewTab.subscribe( url => window.open(url, '_blank'));
```

Comment:
We can also do `window.parent.location.href = url` or `window.top.location.href = url` to load the page in the current url itself. But I find that opening in a new tab is more user friendly since the actual website isn't closed and no context is lost.
P.S Couldn't find a way to do it in elm natively hence the use of port.

Reply:
I've merged some changes here:

https://github.com/parlez-vous/embed/pull/16

Do you mind running the app within the iframe as documented in the README now just to double check that you can do so + to make sure this feature still works as intended.

Reply:
yes, works as intended. opens the link in a new tab and switches to the new tab.

I feel think setup process locally can still be improved, but at least it's much better documented now.

Reply:
p.s rebased the branch onto ur new commit

Reply:
> I feel think setup process locally can still be improved, but at least it's much better documented now.

I definitely agree. I'll create a task on linear for that:

https://linear.app/parlezvous/issue/PAR-78/improve-development-setup-for-embed